src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/DefaultCodeActionProvider.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions
+{
+    internal class DefaultCodeActionProvider : CSharpCodeActionProvider
+    {
+        private static readonly HashSet<Regex> RegexMatchCodeActions = new HashSet<Regex>()
+        {
+            // Supports generating the empty constructor `ClassName()`, as well as constructor with args `ClassName(int)`
+            new Regex(@"Generate constructor '.+\(.*\)'", RegexOptions.IgnoreCase | RegexOptions.Compiled, TimeSpan.FromSeconds(1)),
+
+            // Temporarily disable till we can support multi-part edit formatting
```

Comment:
Is there an issue filed to track this?

Reply:
Filed: https://github.com/dotnet/aspnetcore/issues/26399 thanks.